src/azure-cli/azure/cli/command_modules/monitor/_params.py

```diff
@@ -343,3 +343,9 @@ def load_arguments(self, _):
         c.argument('intelligence_pack_name', options_list=['--name', '-n'])
         c.argument('workspace_name', options_list='--workspace-name')
     # endregion
+
+    # region monitor clone
+    with self.argument_context('monitor clone') as c:
+        c.argument('source_resource', help="Resource ID of the source resource.")
```

Comment:
what is the options_list?

Reply:
`options_list` is used to rename the arguments. I don't want to rename them. Do you think we need a shorter name?